### PR TITLE
featured banner image if available

### PR DESCRIPTION
**Pull request by Lukewh**
Show featured banner images on /store if available.

## QA
- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/store
- See openoffice banner

### CODE REVIEW

File: webapp/store/logic.py [hunk 1]
```diff
@@ -29,6 +29,22 @@ def get_searched_snaps(search_results):
     )
 
 
+def get_snap_banner_url(snap_result):
+    """Get snaps banner url from media object
+
+    :param snap_result: the snap dict
+    :returns: the snap dict with banner key
```
Comment (jpmartinspt) [inline webapp/store/logic.py]: The function is not returning anything!

**Comment by kwm14** [comment]
## Design review
- Nice banner image OnlyOffice 👍 
- Looking good from `$breakpoint-medium` down to mobile @Lukewh 
- But on desktop, the banner is scaling to the correct height but unfortunately, there isn't enough width so leaves us with white space either side

Screenshot 2019-07-29 at 14 04 37

- Removing `display: block;` solves this but still, the banner is too short in length 😢 

Screenshot 2019-07-29 at 14 07 29

**Comment by kwm14** [comment]
## Idea
- Color pick from there banner image which acts as a background color if they upload a smaller size than our recommended `990px` width

Screenshot 2019-07-29 at 14 29 13

**Comment by kwm14** [comment]
## Design convo
_with @Lukewh_ 
- Agreed to test banner left aligned
- Default grey background `$color-light` `#F7F7F7`
- @Lukewh updated in the demo: https://snapcraft-io-canonical-web-and-design-pr-2116.run.demo.haus/
- Need to update the recommended banner size to fit `990px` frame to stop any future situ's like this 👍 
- @therealjuan thoughts?

**Comment by Lukewh** [comment]
- [ ] Scale up to width
- [ ] Contact current banner users
- [ ] Recommend 1920x640 in listings page

**Comment by Lukewh** [comment]
See: https://github.com/canonical-web-and-design/snapcraft.io/pull/2186

**Pull request closed without merging**